src/Model/GContact.php

```diff
@@ -0,0 +1,1004 @@
+<?php
+
+/**
+ * @file src/Model/GContact.php
+ */
+namespace Friendica\Model;
+
+use Friendica\App;
+use Friendica\Core\System;
+use Friendica\Core\Cache;
+use Friendica\Core\Config;
+use Friendica\Core\Worker;
+use Friendica\Database\DBM;
+use Friendica\Network\Probe;
+use Friendica\Protocol\PortableContact;
+use dba;
+
+require_once 'include/datetime.php';
+require_once 'include/network.php';
+require_once 'include/html2bbcode.php';
+require_once 'include/Contact.php';
+require_once 'include/Photo.php';
+
+/**
+ * @brief This class handles global contact related functions
+ */
+class GContact
```

Comment:
Alright, I'm really sorry it didn't occur to me before, but we aren't limited in character for class names, can you expand `GContact` to `GlobalContact` please?

Reply:
done